grails-app/utils/org/pih/warehouse/LocalizationUtil.groovy

```diff
@@ -148,4 +149,16 @@ class LocalizationUtil {
     static String getDefaultString(String str) {
         return getLocalizedString(str, null)
     }
+
+
+    static boolean isSpanishMexico(Locale locale) {
+        return locale.language == "es" && locale.country == "MX"
```

Comment:
I wonder if this could look a bit better, but I guess for now at least I'm ok with it.

Other options:
```groovy
return locale == new Locale("es", "MX")
```
or
```groovy
return locale == LocalizationUtils.getLocale("es-MX") // or es_MX, both works
```

Reply:
sure, if the proper `equals` method is implemented in the `Locale` class, so that your suggestions will work, I'll change it